### PR TITLE
Decloak tooltip

### PR DESCRIPTION
**Pull request by mmazanec22**
- This is a fix that has a bug
- Tooltips show up... but when you resize maps, creates more
- @dm00000 says we can live with this bug for now

### CODE REVIEW

**Comment by jrhalchak** [comment]
I think this is working...

Is it supposed to look like this though?
image

**Comment by mmazanec22** [comment]
Ugh no.  I think it's an issue with mousing over the map before it's completely loaded.  Should I file a bug?  Or just not merge this PR today?

**Comment by jrhalchak** [comment]
@mmazanec22 I seem to get that no matter how many refreshes. 😦

**Comment by csavage1994** [comment]
Cleared the cache and everything? What browser/os are you running?

**Comment by mmazanec22** [comment]
Should look like this

screen shot 2017-03-03 at 11 37 45 am

**Comment by mmazanec22** [comment]
@csavage1994 @jrhalchak I would appreciate some testing of these changes-- I literally just marked all the tooltip CSS as !important, which is garbage, but works

**Comment by mmazanec22** [comment]
Yeah, all of these issues will probably go away if I append the tooltip to its associated map rather than to the body.  Coordinates will be hard, but the other weird problems and hacky fixes can go

**Comment by heronblade** [comment]
looks good to me
🇹🇭 🥇 🌗

**Comment by jrhalchak** [comment]
@mmazanec22 did you actually push your commit with the fix? I don't see it listed on this page after our discussion.

**Comment by heronblade** [comment]
@mmazanec22 wait before you merge

File: public/lib/jvectormap/jquery-jvectormap.css [hunk 1]
```diff
@@ -131,38 +131,38 @@ svg {
...
-    border: 1px solid @5c-charcoal;
-    white-space:nowrap;
-    overflow: hidden;
+    display: flex !important;
+    flex-direction: column !important;
+    border-radius: 0.2em !important;
+    border: 1px solid @5c-charcoal !important;
+    white-space:nowrap !important;
+    overflow: hidden !important;
 }
```
Comment (heronblade) [inline public/lib/jvectormap/jquery-jvectormap.css]: You've got `@` symbols in here but it isn't a `.less` file it's `css`

**Comment by heronblade** [comment]
screen shot 2017-03-03 at 1 45 45 pm

that's what the developer tools are showing

File: public/lib/jvectormap/jquery-jvectormap.css (hunk 1 above)
Comment (jrhalchak) [inline public/lib/jvectormap/jquery-jvectormap.css]: Maybe that's why my console is blowing up and it doesn't work for me!

**Comment by mmazanec22** [comment]
AJ redeployed without it, so f this map for today

File: public/lib/jvectormap/jquery-jvectormap.css (hunk 1 above)
Comment (heronblade) [inline public/lib/jvectormap/jquery-jvectormap.css]: i just assumed it was some black magic that was getting it to work.

**Comment by heronblade** [comment]
When I add hex codes for the colors, and get rid of the `&` rules, I don't need `!important` anymore.

File: public/lib/jvectormap/jquery-jvectormap.css [hunk 2]
```diff
@@ -131,38 +131,38 @@ svg {
...
-    display: flex;
-    flex-direction: column;
-    border-radius: 0.2em;
-    border: 1px solid @5c-charcoal;
-    white-space:nowrap;
-    overflow: hidden;
+    display: flex !important;
+    flex-direction: column !important;
+    border-radius: 0.2em !important;
+    border: 1px solid @5c-charcoal !important;
```
Comment (vforgione) [inline public/lib/jvectormap/jquery-jvectormap.css]: Kill the `@5c-charcoal` and replace it with the right hex value. Calling this out inline for @heronblade

File: public/lib/jvectormap/jquery-jvectormap.css [hunk 3]
```diff
@@ -131,38 +131,38 @@ svg {
...
+    white-space:nowrap !important;
+    overflow: hidden !important;
 }
 
 .tooltipInfoContainer {
-    color: @5c-silver;
-    display: flex;
+    color: @5c-silver !important;
```
Comment (vforgione) [inline public/lib/jvectormap/jquery-jvectormap.css]: Samesies.

File: public/lib/jvectormap/jquery-jvectormap.css [hunk 4]
```diff
@@ -131,38 +131,38 @@ svg {
...
     /*height: 25px;*/
-    flex-shrink: 0;
-    flex: 1;
-    align-items: center;
-    background-color: @5c-rain;
-    padding: 0.25em;
+    flex-shrink: 0 !important;
+    flex: 1 !important;
+    align-items: center !important;
+    background-color: @5c-rain !important;
```
Comment (vforgione) [inline public/lib/jvectormap/jquery-jvectormap.css]: And here.

File: public/lib/jvectormap/jquery-jvectormap.css [hunk 5]
```diff
@@ -131,38 +131,38 @@ svg {
...
+    padding: 0.25em !important;
     &:nth-child(even) {
-        background-color: @5c-rain-dark;
+        background-color: @5c-rain-dark !important;
```
Comment (vforgione) [inline public/lib/jvectormap/jquery-jvectormap.css]: And here.

File: public/lib/jvectormap/jquery-jvectormap.css (hunk 2 above)
Comment (heronblade) [inline public/lib/jvectormap/jquery-jvectormap.css]: thanks :) 👍  🇺🇸

**Comment by mmazanec22** [comment]
Weirdly it worked for me without the correct colors or any !importants, but also... other plugins were styling this tooltip as well as their own maybe?

**Comment by mmazanec22** [comment]
All right, the colors are fixed.

File: public/lib/jvectormap/jquery-jvectormap.css [hunk 6]
```diff
@@ -134,23 +134,19 @@ svg {
...
     flex-shrink: 0;
     flex: 1;
     align-items: center;
-    background-color: @5c-rain;
+    background-color: #343443;
     padding: 0.25em;
-    &:nth-child(even) {
-        background-color: @5c-rain-dark;
-    }
     &:first-of-type {
```
Comment (heronblade) [inline public/lib/jvectormap/jquery-jvectormap.css]: are `&:` nestings legal css? or is that only preprocessors?

Comment (heronblade) [inline public/lib/jvectormap/jquery-jvectormap.css]: are `&:` nestings legal css? or is that only preprocessors?

**Comment by mmazanec22** [comment]
Ok.  At long last, I think this is ready.  Would appreciate some eyes.  @jrhalchak @csavage1994 @heronblade @vforgione

**Comment by mmazanec22** [comment]
@csavage1994 thanks for finding that bug!  Now maybe, finally, it might work.

**Comment by mmazanec22** [comment]
@vforgione now that the css is less wrong, whaddya think?

**Comment by jrhalchak** [comment]
Btw @mmazanec22 you could've kept those variables (I'm assuming you stole them from the branding plugin).  You would've just needed to make it a `.less` file and copied out the variables themselves into a variables file that you import (or just thrown them at the top of the less file).  I think it only compiles relative to the specific plugin you're working in.

**Pull request merged**